bin/pycbc_multi_inspiral

```diff
@@ -211,6 +211,16 @@ def get_projection_matrix(wp, wc):
     return projection_matrix
 
 
+def circular_projection_matrix(wp, wc):
```

Comment:
Can we perhaps unify this into the above function `get_projection_matrix` which could have a kwarg for the projection name?